extensions/wikia/AdEngine/AdEngine2ContextService.class.php

```diff
@@ -66,7 +66,8 @@ public function getContext( Title $title, $skinName ) {
 					'pageName' => $title->getPrefixedDBKey(),
 					'pageType' => $wikiaPageType->getPageType(),
 					'sevenOneMediaSub2Site' => $wg->AdDriverSevenOneMediaOverrideSub2Site,
-					'skin' => $skinName,
+					// TODO: determine mercury skin in the right way
+					'skin' => $skinName === 'oasis' ? 'oasis' : 'mercury',
```

Comment:
Other idea: silly module reused in those two different JS modules which has this silly logic ;) But before we implement it: do we know what will get broken in `wikiamobile` skin?